docs/source/usage/plugins/radiation.rst

```diff
@@ -287,6 +287,10 @@ Command line option                       Description
 ``--<species>_radiation.folderRadPerGPU`` Name of the folder, where the GPU specific spectra are stored.
                                           Default: ``radPerGPU``
 ``--<species>_radiation.compression``     If set, the hdf5 output is compressed.
+``--<species>_radiation.numTmpResults``   Number of temporary results per device.
+                                          This option is used to increase the utilization of the device by producing more independent work.
+                                          Increase results in more independent parallel work and a larger memory footprint.
```

Comment:
This sentence does not seem clear, also grammatically

Reply:
I concur.
A better formulation would be:
"This option enables accumulation of data in parallel into multiple temporary arrays, thereby increasing the utilization of the device by increasing the memory footprint."

Regarding the memory footprint it may be meaningful to add: "The memory usage is increased by $N_{omega}N_{directions}$ for each temporary array."